.gitignore

```diff
@@ -83,11 +83,13 @@ bench-report.xml
 ##############
 
 /enso
+/project-manager
 *.exe
 /enso.exp
 /enso.lib
 *.pdb
 *.jar
+/built-distribution/
```

Comment:
Why not just drop the artifacts into `distribution`, like we do now?

Reply:
The contents of `built-distribution` already have a layout that is ready to be packaged, while `distribution` is currently a bit messy - there is a separate `engine` directory but parts of the `distribution` root directory also belong in `engine` package (like the `std-lib` directory).

I think what you suggested is a good idea but it requires changing the structure of `distribution` directory. I think maybe it should be done as part of #1046?